### PR TITLE
Note about what the performance counters indicate

### PR DESCRIPTION
**Pull request by ramonsmits**
This note seems to be only applicable to V6 and in V5 the performance counters included activity of all queues.

@Particular/nservicebus-maintainers Is that correct?

### CODE REVIEW

File: nservicebus/operations/performance-counters.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ NServiceBus includes several performance counters. They are installed in the `NS
 
 Since all performance counters in Windows are exposed via Windows Management Instrumentation (WMI), it is very straightforward to pull this information into the existing monitoring infrastructure.
 
+NOTE: The NServiceBus performance counters only indicate activity on the logical endpoint queue and endpoint instance specific queue. Any activity on the satellites is not visible.
```
Comment (andreasohlund) [inline nservicebus/operations/performance-counters.md]: Like you mentioned v5 did show satellites as well, should this be split into a v5 and a v6 version?

**Comment by ramonsmits** [comment]
I think this is the way to do it: https://github.com/Particular/docs.particular.net#markdown-partials

**Comment by SimonCropp** [comment]
@ramonsmits can you convert this to a partial?

**Comment by weralabaj** [comment]
@andreasohlund is that what you had in mind ☝️

File: nservicebus/operations/performance-counters_note_(,5].partial.md [hunk 2]
```diff
@@ -0,0 +1 @@
+NOTE: The NServiceBus performance counters only indicate activity on the logical endpoint queue and endpoint instance specific queue. Any activity on the satellites is not visible.
```
Comment (andreasohlund) [inline nservicebus/operations/performance-counters_note_(,5].partial.md]: v5 did show counters for satellites so this is wrong?

Comment (andreasohlund) [inline nservicebus/operations/performance-counters_note_(,5].partial.md]: @ramonsmits can you confirm?

**Pull request merged**